src/pages/LearnPage.tsx

```diff
@@ -117,6 +132,76 @@ export default function LearnPage() {
 
       <SearchBar />
 
+      <LessonGroup title="Intro Algo Design">
+        <InlineSpacing spacing={40} />
+
+        <LessonButton
+          onClick={() =>
+            setLesson(
+              <LessonPage
+                blocks={[
+                  <ReadingBlock content={dp} />,
+                  <MCQBlock
+                    question={q1}
+                    answerOptions={ans1}
+                    correctOptions={correct1}
+                    hint={hint1}
+                    explanation={explanation1}
+                  />,
+                  <MCQBlock
+                    question={q2}
+                    answerOptions={ans2}
+                    correctOptions={correct2}
+                    hint={hint2}
+                    explanation={explanation2}
+                  />,
+                  <ReadingBlock content={markdown1} />,
+                  <ReadingBlock content={markdown2} />,
+                  <ReadingBlock content={markdown3} />,
```

Comment:
Where are you getting these from?

Reply:
@AsirAAlam coded that and it seems like the markdown files are fetched from content repo with Octokit

Reply:
haha no I coded it

but like where are you calling the fetch method? I don't see where these variables are defined for some reason

I must be blind

Reply:
Oh wait, my code is messing up the lessons. I did not realize that we are calling blocks from lessonpage and learnpage is just for animation scroll. Probably messed up when resolving conflicts. Let me fix that.

And yeah thats why you dont see the variables, which are actually in LessonPage.